apps/nuxt3-ssr/components/CohortDetails.vue

```diff
@@ -292,6 +311,16 @@ let fundingAndAcknowledgementItems = computed(() => {
 useHead({ title: cohort?.acronym || cohort?.name });
 
 const messageFilter = `{"filter": {"id":{"equals":"${route.params.cohort}"}}}`;
+
+const crumbs: any = {};
+if (route.params.catalogue) {
+  crumbs[`${route.params.catalogue}`] = `${route.path}/../..`;
+  crumbs["Cohorts"] = `${route.path}/..`;
+} else {
+  crumbs["Home"] = `${route.path}/../../..`;
```

Comment:
silently breaks on route/struct changes , route object is a global var and suggest to be explicit about paths

Reply:
fixed